src/Agent/NewRelic/Agent/Core/Api/AgentApi.cs

```diff
@@ -632,7 +632,7 @@ void work()
             {
                 InternalApi.ErrorFingerprintingCallback(callback);
             }
-            TryInvoke(work, apiName, ApiMethod.ErrorFingerprintingCallback);
+            TryInvoke(work, apiName, ApiMethod.SetErrorGroupCallback);
```

Comment:
In the most recent spec update, the callback name is still ErrorGroupCallback (error_group_callback), so I don't think we should call this "SetErrorGroupCallback".

Reply:
Is the intention of the API to be used as `Agent.ErrorGroupCallback(MyType.MyCallback)` or `Agent.ErrorGroupCallback = MyType.MyCallback`? Given the name, I think a property syntax makes the most sense, in which case the `Set` is implied. The method form is kind of confusing to me.

Reply:
Ah, I took that to mean the name of the parameter, not the API call. It does look like the spec typically calls out the function name. I guess I should have raised this then. :)

Reply:
We discussed this in pairing and decided to go with Set...